codetransformer/transformers/precomputed_slices.py

```diff
@@ -0,0 +1,20 @@
+"""
```

Comment:
I think of `asconstants` as being about injecting your own custom constants.

I'd be more inclined to put this in something like a top-level `optimizers` package that has transformers that do semantics-preserving optimizaitons.